artifacts/deployment/README.md

```diff
@@ -0,0 +1,31 @@
+# `EtcdProxyController` manifest
+
+This directory contains manifest for deploying the `EtcdProxyController` in the cluster. It creates the following resources:
+* **namespace/kube-apiserver-storage** - namespace where controller and resources managed by controller are deployed.
+* **serviceaccount/etcdproxy-controller-sa** - ServiceAccount used by Controller for managing EtcdStorage objects, ReplicaSets and Services.
+* **serviceaccount/etcdproxy-sa** - ServiceAccount used by EtcdProxy Pods.
+* **clusterrole/etcdproxy-crd-clusterrole** - ClusterRole allowing **etcdproxy-controller-sa** ServiceAccount to update EtcdStorage status.
+* **role/etcdproxy-controller-role** - Role allowing **etcdproxy-controller-sa** ServiceAccount to manage ReplicaSets, Services, ConfigMap and Secrets in the **kube-apiserver-storage** namespace.
+* **role/etcdproxy-role** - Role allowing **etcdproxy-sa** (EtcdProxy Pods) ServiceAccount to read the ConfigMap and Secret with the core etcd certificates, in the controller's namespace.
+* **clusterrolebinding/etcdproxy-crd-clusterrolebinding** - Binds **clusterrole/etcdproxy-crd-clusterrole** to **serviceaccount/etcdproxy-controller-sa**.
+* **rolebinding/etcdproxy-controller-rolebinding** - Binds **role/etcdproxy-controller-role** to **serviceaccount/etcdproxy-controller-sa**.
+* **etcdproxy-rolebinding** - Binds to **role/etcdproxy-role** to **serviceaccount/etcdproxy-sa**.
+* **customresourcedefinition/etcdstorages.etcd.xmudrii.com** - EtcdStorage CRD used to manage etcd proxies.
+* **deployment/etcdproxy-controller-deployment** - Deployment for the `EtcdProxyController`.
+
+## RBAC Rules
+
+The **clusterrole/etcdproxy-crd-clusterrole** allows only updates to the Status field of the EtcdStorage CustomResource.
+The role is supposed to be bound to the Controller's ServiceAccount.
+
+The **role/etcdproxy-controller-role** ensures Controller can manage ReplicaSets and Services. The role is supposed to be bound to the Controller's Service Account.
+
+The **role/etcdproxy-role** ensures EtcdProxy Pods can only read ConfigMap and Secret containing the certificates for the core etcd.
```

Comment:
I don't think that you need this because they can be mounted.

Reply:
Updated.